modules/ROOT/pages/coverage-studio-concept.adoc

```diff
@@ -6,51 +6,67 @@ endif::[]
+                number of event processors in the Mule configuration file
+| *Coverage*    | Percentage of event processors in the Mule configuration file that the MUnit test executes
 |===
 
-Additionally you can click in each Resource, to get a more specific and granular report of the Event Processors executed in that resource:
+Additionally, you can click on each *Resource* to get a more specific and granular report of the Event Processors executed in that resource:
 
-image::munit-granular-coverage-report-in-anypoint-studio.png[]
+image::munit-granular-coverage-report-in-anypoint-studio.png["MUnit Coverage Report with the Granular report highlighted"]
 
 [%header,cols="20a,80a"]
 |===
 | Column              | Description
-| *Name*                | The name of the container.
-| *Type*                | The type of the container.
-| *Covered Processors* | The processors inside the container that were executed during the MUnit test.
-| *Processors*         | The total amount of processors inside the container.
-| *Coverage*            | This field represents the percentage of the event processors inside the container that were executed in the MUnit test.
+| *Name*                | Name of the container
+| *Type*                | Type of the container
+| *Covered Processors* | Processors inside the container that are executed during the MUnit test
+| *Processors*         | Total amount of processors inside the container
+| *Coverage*            | Percentage of the event processors inside the container executed in the MUnit test
 |===
 
 == Disable Coverage Reports
 
-If you choose so, you can disable the coverage calculation from your project's run configuration.
+If you choose so, you can disable the coverage calculation from your project run configuration so you test your flow without a coverage report.
 
-Right click in your Package Explorer, and choose *Run As* and *Run Configurations...*, select your MUnit run configuration file to access your suite's configuration, and deselect the "Run coverage" option under your coverage settings.
+Right-click *Package Explorer* and choose *Run As* > *Run Configurations*. Select your MUnit run configuration file to access your suite configuration, and deselect the *Run coverage* option in your coverage settings.
 
-image::disable-munit-coverage-in-anypoint-studio.png[]
+image::disable-munit-coverage-in-anypoint-studio.png["MUnit run and Run coverage highlighted"]
+
+== Exclude XML Files from MUnit Coverage Report
+
+You can exclude one or more XML files from an MUnit Coverage report when testing your application. 
+
+. Go to the flow to change.
+. Click *Run* > *Run Configuration*.
+. Click the name of your project.
+. In the *Ignored files* section, click *Add*.
++
+image::add-ignored-files.png["Add Ignored File window highlighted in the Test tab"]
++
+. Type the name of the XML file to exclude from the Coverage report and click *OK*.
+. Click *Apply* and *Run*.
+
+== See Also
```

Comment:
Add a link to this one too?
https://docs.mulesoft.com/munit/latest/munit-coverage-report

Reply:
Sure! Thanks!